### ISSUE TITLE
Warn when running unelevated on Windows

### ISSUE THREAD

**Issue by oliversalzburg** [bloglovin/lintlovin#16]
On Windows OS, it is usually not possible to create symlinks, unless you're running in an elevated shell (which is uncommon).

I would recommend a combination of [is-windows](https://www.npmjs.com/package/is-windows) and [is-administrator](https://www.npmjs.com/package/is-administrator) to detect the situation and warn accordingly.

**Comment by voxpelli** [comment]
@oliversalzburg: Warn so that the user can fix it itself? If so it should probably advice the user on how to fix it as well. I've no idea on how to run an "elevated shell" though.

(It should also check that the symlinked files hasn't been overridden in the repo as that's sometimes something that happens)

Maybe lintlovin can provide an automatic fallback if it detects that the files are supposed to be symlinks but that the symlinks failed? That way Windows users wouldn't have to do any work.

Symlinks are by default checked out as a file containing the path to the symlinked file on Windows – right? So if `is-windows` returns `true` one could check whether the possibly symlinked files are such files and if so reconfigure the Grunt tasks to use the actual files instead.

**Comment by oliversalzburg** [comment]
Yeah, I had the files with the path to the linked file in them when I checkout out the repo. Quite confusing at the time.

Sure, the user could be instructed how to solve the problem. You have to run Command Prompt or Git Bash with the "Run as Administrator" context menu option. Windows developers should be well aware of the process. From such a shell, all the regular symlinking operations work as expected.

I like the fallback idea, although the only fallback I can think of right now, would be to copy the files.

**Comment by voxpelli** [comment]
@oliversalzburg The Grunt-configuration explicitly points out most of the config files: https://github.com/bloglovin/lintlovin/blob/master/index.js#L39 So it could reconfigure it if it finds the need
